ddtrace/helpers.py

```diff
@@ -10,7 +10,7 @@
     from ddtrace.tracer import Tracer
 
 
-@deprecated("This method and module will be removed altogether", "1.0.0")
+@deprecated("This method and module will be removed altogether. Use 'ddtrace.Tracer.get_log_correlation_context()' instead.", "1.0.0")
```

Comment:
Fix black formatting.

```suggestion
@deprecated(
    "This method and module will be removed altogether. Use 'ddtrace.Tracer.get_log_correlation_context()' instead.",
    "1.0.0",
)
```